packages/core/src/shared/clients/codecatalystClient.ts

```diff
@@ -175,7 +178,16 @@ export async function createClient(
 ): Promise<CodeCatalystClient> {
     const sdkClient = await createCodeCatalystClient(connection, regionCode, endpoint, retryOptions)
     const c = new CodeCatalystClientInternal(connection, sdkClient)
-    await c.verifySession()
+    try {
+        await c.verifySession()
+    } catch (e) {
+        if (!(e instanceof ToolkitError) || e.code !== 'AccessDeniedException') throw e
+        onAccessDeniedExceptionEmitter.fire()
+
+        // We "cancel" this error so that it prevents further execution,
+        // but doesn't display the error
```

Comment:
The wording is confusing. We aren't cancelling an error, but cancelling execution, right?
```suggestion
        // Throw a "cancel" error to prevent further execution.
```